README.md

```diff
@@ -109,6 +109,10 @@ sudo make install
 NOCONFIGURE=true ./autogen.sh
 mkdir build; cd build
 ../configure
+make
+sudo make install
+sudo cp -v data/com.github.gnotime.gnotime.gschema.xml /usr(/local)/share/glib-2.0/schemas/
+sudo glib-compile-schemas /usr(/local)/share/glib-2.0/schemas/
```

Comment:
Is this what you meant by not getting it working with automake?

> Just adding the respective lines to Makefile.am as suggested by the old Gnome documentation ([link](https://developer-old.gnome.org/GSettings)) was without effect on my tries.

It seems a more modern version of the instructions you linked to are at https://wiki.gnome.org/HowDoI/GSettings, but the contents regarding automake is the same.

Just to confirm, did you notice it says "Makefile.am in *the same directory as the schema file*"?

Reply:
That page also says:

> Typically, you will place this file in the toplevel data/ directory in your project's source, but you can put it anywhere you like, according to the conventions used in your project. 

The way they worded that, it sounds like they assume that the project already has a data directory.

To add one, and make automake pick up on it I had to:

   * Add `data` to `SUBDIRS` in the top-level Makefile.am.
   * Add `data/Makefile` to `AC_OUTPUT` in configure.in (towards the end).

The last bit was inspired by (https://www.gnu.org/software/automake/manual/automake.html#index-AC_005fCONFIG_005fFILES), although we use a deprecated style where the parameter is for AC_OUTPUT instead of AC_CONFIG_FILES (https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.71/autoconf.html#index-AC_005fOUTPUT-3).

This appears to make it work using the regular autogen/configure/make/install process, but I can't say for sure if this is all there's to it. I'm not fluent in automake :)

Reply:
You must be more fluent than me :-D I had all bits together, but I did not know that I have to add a reference to `AC_OUTPUT` too. Now with your hint it works as expected and install the schema file automatically, thank you :-)

I dropped my change from the `README.md`, since it's not needed anymore.